x/emissions/proto/emissions/v1/node.proto

```diff
@@ -4,6 +4,9 @@ package emissions.v1;
 option go_package = "github.com/allora-network/allora-chain/x/emissions/types";
 
 message OffchainNode {
-  string node_address = 1;
-  string owner = 2;
+  string lib_p2p_key = 1;    // LibP2P key of the node
+  string multi_address = 2;  // Network address for accessing the node
+  string owner = 3;
+  string node_address = 4;
+  string node_id = 5;
```

Comment:
To use reserved we would need to delete the fields. Can we delete the fields? Is there any client code using this, or it's just used internally? 

Deprecated only signals to clients that they should not use the field because it might get removed in the future.

Reply:
with the "b7s removal" project we're updating the only "allora official" client, and it's prehistory in our ecosystem's history so not too worried about alternative community clients. valid concern otherwise